buildenv/jenkins/JenkinsfileBase

```diff
@@ -1026,7 +1036,7 @@ def run_parallel_tests() {
 					currentBuild.result = buildResult
 				}
 				archiveFile("**/*.tap")
-				addFailedTestsGrinderLink()
+				addFailedTestsGrinderLink(buildPaths.substring(0, buildPaths.length() - 1))
```

Comment:
Please check `buildPaths` contains the value. Otherwise, we will get IndexOutOfBoundsException.